packages/datadog-plugin-aws-sdk/src/services/eventbridge.js

```diff
@@ -26,19 +26,32 @@ class EventBridge {
     if (operation === 'putEvents' &&
       request.params &&
       request.params.Entries &&
-      request.params.Entries.length > 0 &&
-      request.params.Entries[0].Detail) {
+      request.params.Entries.length > 0) {
       try {
-        const details = JSON.parse(request.params.Entries[0].Detail)
-        details._datadog = {}
-        tracer.inject(span, 'text_map', details._datadog)
-        const finalData = JSON.stringify(details)
-        const byteSize = Buffer.byteLength(finalData)
-        if (byteSize >= (1024 * 256)) {
-          log.info('Payload size too large to pass context')
-          return
-        }
-        request.params.Entries[0].Detail = finalData
+        // eslint-disable-next-line no-console
+        console.log('AGOCS 1')
+        const currentMilliseconds = new Date().getMilliseconds()
```

Comment:
Is it safe to grab just the milliseconds here? Since this is happening later I'm not sure it would be accurate. Any way to replace the entire date/time with `Date.now()`?

Reply:
This is a good question. Let me see if there's anything on Amazon's side preventing us from doing that.

Reply:
I noodled on this for a minute. The user is allowed to supply their own [time](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-eventbridge/modules/puteventsrequestentry.html#time), and realistically, we have no way of knowing if the time we got is from the user or from the AWS Javascript SDK. I think the best we can do is add our own milliseconds if the supplied time is not at millisecond resolution.

I worried about accuracy too, but on further reflection, our `requestInject` will be called nanoseconds or milliseconds after the user calls `PutEvents`, so the difference will be negligible.